src/Apps/oc_outfits.lsl

```diff
@@ -218,7 +247,7 @@ UserCommand(integer iNum, string sStr, key kID) {
             RmCorelock();
             llSleep(1);
         }
-        if(sChangetype == "wear"){
+        if(sChangetype == "wear"){ //This looks like dead code TODO: Verify for removal?
```

Comment:
nope not dead code, the block above it performs the removal actions if applicable, then this block will perform the add operation